.gitignore

```diff
@@ -1,7 +1,7 @@
 node_modules
 build
-results.xlsx*
-temp.xlsx
+*.xlsx
 *.log
 db/
+data/
```

Comment:
Is this data folder associated with mongoDB?

Reply:
A relic of my local setup which used the data/ folder as the DB folder